pkg/controllers/deprovisioning/consolidation.go

```diff
@@ -93,17 +90,17 @@ func (c *consolidation) markConsolidated() {
 }
 
 // ShouldDeprovision is a predicate used to filter deprovisionable nodes
-func (c *consolidation) ShouldDeprovision(_ context.Context, cn *Candidate) bool {
-	if val, ok := cn.Annotations()[v1alpha5.DoNotConsolidateNodeAnnotationKey]; ok {
-		c.recorder.Publish(deprovisioningevents.Unconsolidatable(cn.Node, cn.Machine, fmt.Sprintf("%s annotation exists", v1alpha5.DoNotConsolidateNodeAnnotationKey))...)
-		return val != "true"
+func (c *consolidation) shouldDeprovision(_ context.Context, cn *Candidate, isEmptyConsolidation bool) bool {
+	if cn.Annotations()[v1alpha5.DoNotConsolidateNodeAnnotationKey] == "true" {
```

Comment:
Just musing: it would be nice to rename this to `safe-to-consolidate` similar to `safe-to-disrupt` when we move it over.

Reply:
We aren't planning to maintain this label long-term. The idea was to remove this label in favor of `do-not-disrupt` to stop all voluntarily disruption (since that's the use-case). This is there for compat and should be pulled out long-term.